src/containerapp/azext_containerapp/commands.py

```diff
@@ -213,7 +213,7 @@ def load_command_table(self, args):
     with self.command_group('containerapp job replica', is_preview=True) as g:
         g.custom_show_command('list', 'list_replica_containerappsjob')
 
-    with self.command_group('containerapp env java-component nacos') as g:
+    with self.command_group('containerapp env java-component nacos', is_preview=True) as g:
```

Comment:
Why update `az containerapp env java-component nacos` to preview?

Reply:
The nacos should be in preview status, and the preview is removed from java component level, we need add the preview to nacos level. Just add in this pr for convenience